<?xml version="1.0"?>
<launch>
    <arg name="instance_topic_name" default="kansei_pub"/>
    <arg name="dummy_topic_name" default="instance_pub"/>
    
    <node pkg="estimator" type="color_pointcloud" name="color_pointcloud" output="screen">
        <param name="instance_topic_name" value="$(arg instance_topic_name)"/>
        <param name="dummy_topic_name" value="$(arg dummy_topic_name)"/>
    </node>
</launch>